src/Database/Log/QueryLogger.php

```diff
@@ -36,6 +43,9 @@ public function __construct(array $config = [])
     {
         $this->_defaultConfig['scopes'] = ['queriesLog'];
 
+        $this->connection = $config['connection'] ?? '';
```

Comment:
Why use a separate property and unset `'connection'` key instead of just using `$this->getConfig('connection')` later on?

Reply:
I was trying to keep the overhead low, but it's probably minimal.

Reply:
I simplified it down to getting the config value.